inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/WorkflowApi.java

```diff
@@ -37,8 +43,29 @@ public interface WorkflowApi {
     @POST("workflow/approve/{taskId}")
     Call<Response<WorkflowResult>> startInlongGroup(@Path("taskId") Integer taskId, @Body Map<String, Object> request);
 
-    @GET("workflow/event/list")
-    Call<Response<PageInfo<EventLogResponse>>> getInlongGroupError(@Query("inlongGroupId") String groupId,
-            @Query("status") Integer status);
+    @POST("workflow/start")
+    Call<Response<WorkflowResult>> start(@Body WorkflowOperationRequest request);
+
+    @POST("workflow/cancel/{taskId}")
+    Call<Response<WorkflowResult>> cancel(@Path("processId") Integer processId, @Body WorkflowOperationRequest request);
+
+    @POST("workflow/continue/{taskId}")
+    Call<Response<WorkflowResult>> continueProcess(@Path("processId") Integer processId,
+            @Body WorkflowOperationRequest request);
+
+    @POST("workflow/reject/{taskId}")
+    Call<Response<WorkflowResult>> reject(@Path("taskId") Integer taskId, @Body WorkflowOperationRequest request);
+
+    @POST("workflow/complete/{taskId}")
+    Call<Response<WorkflowResult>> complete(@Path("taskId") Integer taskId, @Body WorkflowOperationRequest request);
+
+    @GET("workflow/detail/{id}")
+    Call<Response<ProcessDetailResponse>> detail(@Path("processId") Integer processId, @Query("taskId") Integer taskId);
+
+    @GET("workflow/listProcess")
```

Comment:
I remember Get doesn't use objects, please confirm. If not, you need to use Map to receive QueryParam

You can refer this ISSUE https://github.com/square/retrofit/issues/2293#issuecomment-683286832

https://github.com/square/retrofit/issues/2293#issuecomment-683286832

Reply:
Yeah, the [user API](https://github.com/apache/inlong/pull/5494/files) has the same problem, I will fix it later.